backend/protocol_rpc/endpoints.py

```diff
@@ -255,25 +255,23 @@ def get_transaction_by_id(
 
 def get_contract_state(
     accounts_manager: AccountsManager,
-    msg_handler: MessageHandler,
     contract_address: str,
     method_name: str,
     method_args: list,
 ) -> dict:
-    if not accounts_manager.is_valid_address(contract_address):
+    if not address_is_in_correct_format(contract_address):
         raise InvalidAddressError(contract_address)
 
-    contract_account = accounts_manager.get_account(contract_address)
+    contract_account = accounts_manager.get_account_or_fail(contract_address)
     node = Node(
         contract_snapshot=None,
         address="",
-        validator_mode=ExecutionMode.LEADER,
+        validator_mode="leader",
```

Comment:
@AgustinRamiroDiaz why is it better as a hardcoded string rather than an enum?

Reply:
this was a problem with merge conflicts. Fixed!